bindings/java/wasmedge-java/wasmedge-jni/lib/GlobalTypeContext.c

```diff
@@ -35,10 +62,39 @@ Java_org_wasmedge_GlobalTypeContext_close(JNIEnv *env, jobject thisObject) {
 
 JNIEXPORT jint JNICALL Java_org_wasmedge_GlobalTypeContext_nativeGetValueType(
     JNIEnv *env, jobject thisObject) {
-  return WasmEdge_GlobalTypeGetValType(getGlobalTypeContext(env, thisObject));
+  WasmEdge_GlobalTypeContext *globalTypeContext = getGlobalTypeContext(env, thisObject);
```

Comment:
I would say if this is no longer been used by any internal works here, then remove this, since user should invoke `IsXXX` to check to avoid leak more details of implementation.